docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    discord_nn_idx = mp[discord_idx, 1]  # discords nearest neighborhood index
</pre><p> </p><p>Regarding  the comment: <code>discord nearest neighbor index</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>